README.md

```diff
@@ -2,21 +2,122 @@
+needs to be revisited in the future. Next, it facilitates asynchronous conversation from
+a geographically distributed user-base. To that end, any discussion on the teleconference
+about specific Issues or Pull Requests will be noted as a comment to that Issue or Pull Request. Lastly, the GitHub mechanism also serves as our consensus building medium for accepting changes
+once all dissenting opinions and questions have been resolved, providing an archive of
+the responses to those concerns and those individuals participating in the process.
 
-### Participating in the discussion
 
-If you have a question about the PMIx interface or want to raise a question for discussion then please file an Issue against this repository. We are using the Issues to discuss topics, and will link Pull Requests related to that topic to that Issue. This provides the community with a public, archived forum for discussion.
+## Standardization Process
 
-### Participatin in the writing
+The PMIx standardization process considers contributions by using the general workflow outlined below:
 
-If you want to write a portion of the standard (or even fix a typo or two), then use the Pull Request mechanism to post the change for review.
+ 1. Open a **GitHub Issue** describing your contribution for general discussion.
+ 2. Open a **GitHub Pull Request (PR)** if a specific change to the PMIx Standard is needed.
+    * Reference the corresponding Issue(s). Keep the PR marked as [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) until an implementation is available and the PR is ready for broader discussion.
+    * Send an email to the [PMIx Forum mailing list](https://groups.google.com/forum/#!forum/pmix-forum) once the PR is ready for discussion.
+    * Once the discussion has resolved (i.e., dissenting opinions and questions have been addressed) then it can move to an official "Reading" in the next teleconference. Email the mailing list once it is ready for "Reading" to have it added to the teleconference agenda.
+ 3. Official "**Reading**" of the PR during a teleconference
+    * If no revisions are required or objections raised (either on the PR or in the teleconference) then the PR can be considered for "Acceptance" in the next teleconference.
+ 4. Official "**Acceptance**" of the PR during a following teleconference
+    * If no objection is raised (either on the PR on in the teleconference) then the PR is merged into the document and, if also resolved, the corresponding Issue is closed.
+
+**General Statements:**
+
+ * Any individual or organization may participate in the discussion on a topic.
+ * Regular teleconferences are announced on the PMIx Forum mailing list
+   - Teleconferences provide a high bandwidth discussion format for various topics
+     surrounding the PMIx Standard, and a synchronization mechanism to guide progress.
+   - Notes from the teleconferences will be archived on the PMIx Forum wiki.
```

Comment:
Do we have a link to the wiki?  Is it this: https://github.com/pmix/pmix-standard/wiki#meeting-information?

Reply:
Yeah I'll embed that link.

Reply:
Fixed